data/hai/unfettered jobs.txt

```diff
@@ -324,13 +324,14 @@ mission "Unfettered Small Medicals Rush"
 		attributes "spaceport"
 	destination "Darkcloak"
 	to offer
-		random < 30 + "Unfettered Jump Drive 1: offered" * 30
+		random < 20 + "Unfettered Jump Drive 2: offered" * 20
```

Comment:
```suggestion
		random < 40 + "Unfettered Jump Drive 2: offered" * 20
```

Reply:
```suggestion
		random < 20 + "Unfettered Jump Drive 1: offered" * 20 "Unfettered Jump Drive 2: offered" * 20
```
this doesnt change anything in the current game compared to what you proposed but would have an impact once hai legacy is merged (as it gives positive rep without a JD)